pystiche_papers/johnson_alahi_li_2016/_modules.py

```diff
@@ -262,6 +289,7 @@ def __init__(
             self.init_weights()
 
     def init_weights(self) -> None:
+        r"""Initializes the weights of the transformer as it is the case in the original implementation."""
         for module in self.modules():
             if isinstance(module, (nn.Conv2d, nn.ConvTranspose2d)):
```

Comment:
Please add a link to the original luatorch initalization scheme for these layers.